jobs/etcd/spec

```diff
@@ -56,9 +58,29 @@ properties:
     description: Certificate for etcdctl client authentication
   tls.etcdctl.private_key:
     description: Private key for etcdctl client authentication
+  tls.etcdctl-flanneld.certificate:
```

Comment:
Do we even need these? Flannel currently consumes certs via the `etcd` link, but given that `etcd` doesn't appear to need to know of these (based on the property not being used), there's no reason we couldn't just pass these creds directly to Flannel.

Reply:
Cool let me try